samples/bluetooth/unicast_audio_server/src/main.c

```diff
@@ -25,12 +25,20 @@ NET_BUF_POOL_FIXED_DEFINE(tx_pool, 1, CONFIG_BT_ISO_TX_MTU, NULL);
 static struct bt_conn *default_conn;
 static struct bt_audio_stream streams[MAX_PAC];
 
+
+static uint8_t unicast_server_addata[] = {
+	0x4E, 0x18, /* ASCS UUID */
+	0x01, /* Target Announcement */
+	0xFF, 0x00, 0xFF, 0x00, /* Available Audio Context */
```

Comment:
We should add `#define` for these values if they don't exist.

Reply:
Yes, I agree that add #define for these Generic Audio Context Type, furthermore, Target/General Announcement. Which files do you think are suitable for bitmask definitions?

Reply:
It is a good question. We have a lot of missing definitions for LE Audio, and some of them are defined the profile specs or service specs, and some of them are defined by the Generic Audio Assigned Numbers. 

For now I would just put them in the audio.h file, and then possibly we'll move the BAP-only stuff from that file into other header files to clean it up (it is already very large).

Reply:
> It is a good question. We have a lot of missing definitions for LE Audio, and some of them are defined the profile specs or service specs, and some of them are defined by the Generic Audio Assigned Numbers.
> 
> For now I would just put them in the `audio.h` file, and then possibly we'll move the BAP-only stuff from that file into other header files to clean it up (it is already very large).

OK, I will put the definitions in audio.h temporarily.